samples/httpbin/sample-client/fortio-deploy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: fortio-deploy
+spec:
```

Comment:
can you add
```
  ports:
  - port: 8080
    name: http-fortio
  - port: 8079
    name: grpc-ping
```
that way it can be used also as server if needed/wanted